float64 iou
float64 confidence
bool classification